arches_controlled_lists/src/arches_controlled_lists/components/editor/ListSearchable.vue

```diff
@@ -55,7 +55,11 @@ const save = async () => {
 <template>
     <div class="characteristic">
         <div class="value-editor-title">
-            <h4>{{ $gettext("Searchable?") }}</h4>
+            <label
+                for="searchableSwitch"
+                class="h4"
+                >{{ $gettext("Searchable?") }}</label
+            >
```

Comment:
Prettier chewed this up, mind hand formatting?

Reply:
done!